src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -35,14 +33,13 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
 
     @action componentDidMount() {
         if (this.settingsFormKey) {
-            const {formInspector} = this.props;
-
+            // initialize empty blockSettingsFormStore because schema of the store is used for determining iconsMapping
             this.blockSettingsFormStore = memoryFormStoreFactory.createFromFormKey(
```

Comment:
unfortunately we need to initialize an empty `blockSettingsFormStore` because the schema of the store is used to determine which icons are displayed for the block in the `iconsMapping` getter

Reply:
okay then it is that case its sad but we can not currently do it aslong as this lives in the FieldBlocks.js